File: java/uri-online-judge/1009.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.text.DecimalFormat;

public class Main {
  public static void main(String[] args) throws IOException {
    BufferedReader stdin = new BufferedReader( new InputStreamReader( System.in ) );
    DecimalFormat dc = new DecimalFormat("0.00");

    String name      = stdin.readLine();
    double salary    = Double.parseDouble( stdin.readLine() );
    double totalSold = Double.parseDouble( stdin.readLine() );

    double commissionRate = 0.15;
    double commission = totalSold * commissionRate;

    double totalSalary = salary + commission;

    System.out.println("TOTAL = R$ " + dc.format(totalSalary));
  }
} 

File: c++/uri-online-judge/1009.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
  string name;
  double salary, totalSold;

  cin >> name >> salary >> totalSold;
  cout << setprecision(2) << fixed << "TOTAL = R$ " << (salary + totalSold * 0.15) << endl;
  return 0;
}